### ISSUE TITLE
How to trian with SSD-Net

### ISSUE THREAD

**Issue by Mrils** [HiKapok/RON_Tensorflow#3]
Hello, HiKapok!
I'm so glad to find out your codes to fix up the loss problem which occurs in SSD-Tensorflow. But I wonder how to train the vgg-SSD-Net by your codes. Should I write a new script like ron_net_multi_gpu.py?

**Comment by HiKapok** [comment]
@Mrils I'm sorry for that I won't have the time to modify the SSD-Net by myself, you can follow ron_net.py or just use the codes of SSD_Tensorflow but fix those bugs I have mentioned in README.md

**Comment by HiKapok** [comment]
By the way, I will write a new SSD in TensorFlow to get better performance before this month. You can check my repo then if you are interested in. Thank you.

**Comment by Mrils** [comment]
That's all right. Thank you for your response. I'll try to fix these bugs mentioned in your README.md by myself and looking forward to your works. Thank you.

**Comment by HiKapok** [comment]
New SSD yield state-of-the-art performance [here](https://github.com/HiKapok/SSD.TensorFlow)
